### PR TITLE
Add new styleguide tab in storybook

### PR DESCRIPTION
**Pull request by KatieOfCode**
Added a new styleguide tab in the Storybook instance so that contributors can easily see class names and how the component renders with that specific class. There are also notes for when to use the different buttons. Please let me know if there is anything else I can help with. 

Here is the issue this is referring to: https://github.com/ifmeorg/ifme/issues/1149

### CODE REVIEW

File: config/env/development.example.env [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Application
-SECRET_KEY_BASE=""
-DEVISE_SECRET_KEY=""
+SECRET_KEY_BASE="7ba87bf80efaf54543aaa13f1a83cc598944c2c56411a6657c380156cdf2ef2950ffbbee5d3ae8ac07c65aaa12465b37ce5bbfc905f2adcc8ff18d481f29838d"
```
Comment (julianguyen) [inline config/env/development.example.env]: Looks like you accidentally committed this! These values should be stored in `development.env`

File: config/env/test.example.env [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Application
-SECRET_KEY_BASE=""
-DEVISE_SECRET_KEY=""
+SECRET_KEY_BASE="7ba87bf80efaf54543aaa13f1a83cc598944c2c56411a6657c380156cdf2ef2950ffbbee5d3ae8ac07c65aaa12465b37ce5bbfc905f2adcc8ff18d481f29838d"
+DEVISE_SECRET_KEY="f87a7bcf0ed12b1bf931d320167022f71aab151dbf68cec13ecac75c1fa1f2a62d8a29890a3bd691dc0a4e86cae661872cbbdb7e4216948e4412f7ee33737330"
```
Comment (julianguyen) [inline config/env/test.example.env]: Looks like you accidentally committed this! These values should be stored in `test.env`

File: client/app/stories/Styleguide.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,358 @@
+import React, {Fragment} from 'react';
```
Comment (julianguyen) [inline client/app/stories/Styleguide.stories.jsx]: Let's fix the indentation - it should be 2 space = 1 tab.

File: client/app/stories/Styleguide.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+      style={{
+        display: 'inline',
+        Right: '10px',
+        margin: '0 auto',
+        textAlign: 'center',
+        backgroundColor: 'transparent',
+        paddingBottom: '10px',
+      }}
+    >
+      <div
```
Comment (julianguyen) [inline client/app/stories/Styleguide.stories.jsx]: Looks like this style exists in multiple places here, line 46, and line 75. 

We can do something like this:
```
const buttonsGroupStyle = {
  display: 'inline',
  right: '10px',
  margin: '0 auto',
  textAlign: 'center',
  backgroundColor: 'transparent',
  paddingBottom: '10px',
};
```

And then we can do `<div style={buttonsGroupStyle}>`

File: client/app/stories/Styleguide.stories.jsx [hunk 5]
```diff
@@ -0,0 +1,358 @@
...
+        </div>
+      </div>
+    </Fragment>
+  ), { notes: { markdown: '# hi' } })
+  .add('Margins', () => (
+    <Fragment>
+      <div style={{
+        margin:'10px'
+        }}>
+        <div className={`${css.marginRight} marginRight`} style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}>
```
Comment (julianguyen) [inline client/app/stories/Styleguide.stories.jsx]: We can store the following `style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}` as:
```
const marginGroupStyle = {
  display: 'inline-block',
  border: '1px solid',
  padding:'10px',
};
```

```
<div className={`${css.marginRight} marginRight`} style={marginGroupStyle}>
```

File: client/app/stories/Styleguide.stories.jsx [hunk 6]
```diff
@@ -0,0 +1,358 @@
...
+          marginRight
+        </div>
+        <div style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}>
+          marginRight
+        </div>
+      </div>
+      <div style={{
+        margin:'10px'
+        }}>
+        <div className={`${css.smallMarginRight} smallMarginRight`} style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}>
```
Comment (julianguyen) [inline client/app/stories/Styleguide.stories.jsx]: Then you can do:
```
<div className={`${css.smallMarginRight} smallMarginRight`} style={marginGroupStyle}>
```

File: client/app/stories/Styleguide.stories.jsx [hunk 7]
```diff
@@ -0,0 +1,358 @@
...
+  ), { notes: { markdown: '# hi' } })
+  .add('Margins', () => (
+    <Fragment>
+      <div style={{
+        margin:'10px'
+        }}>
+        <div className={`${css.marginRight} marginRight`} style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}>
+          marginRight
+        </div>
+        <div style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}>
```
Comment (julianguyen) [inline client/app/stories/Styleguide.stories.jsx]: Then you can do:
```
<div style={marginGroupStyle}>
```

File: client/app/stories/Styleguide.stories.jsx [hunk 8]
```diff
@@ -0,0 +1,358 @@
...
+          marginRight
+        </div>
+      </div>
+      <div style={{
+        margin:'10px'
+        }}>
+        <div className={`${css.smallMarginRight} smallMarginRight`} style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}>
+          smallMarginRight
+        </div>
+        <div style={{ display: 'inline-block', border: '1px solid', padding:'10px'}}>
```
Comment (julianguyen) [inline client/app/stories/Styleguide.stories.jsx]: Then you can do:
```
<div style={marginGroupStyle}>
```

File: client/app/stories/Styleguide.stories.jsx [hunk 9]
```diff
@@ -0,0 +1,358 @@
...
+          smallMarginRight
+        </div>
+      </div>
+      <div style={{
+        margin:'10px'
+       }}>
+        <div className={`${css.marginLeft} marginLeft`} style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}>
```
Comment (julianguyen) [inline client/app/stories/Styleguide.stories.jsx]: Then you can do:
```
<div className={`${css.marginLeft} marginLeft`} style={marginGroupStyle}>
```

File: client/app/stories/Styleguide.stories.jsx [hunk 10]
```diff
@@ -0,0 +1,358 @@
...
+          marginLeft
+        </div>
+      </div>
+      <div style={{
+        margin:'10px',
+        }}>
+        <div className={`${css.smallMarginLeft} smallMarginLeft`} style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}>
```
Comment (julianguyen) [inline client/app/stories/Styleguide.stories.jsx]: Then you can do:
```
<div className={`${css.smallMarginLeft} smallMarginLeft`} style={marginGroupStyle}>
```

File: client/app/stories/Styleguide.stories.jsx [hunk 11]
```diff
@@ -0,0 +1,358 @@
...
+  .add('Colors', () => (
+    <Fragment>
+      <div
+        className={`${css.gridThree} gridThree`}
+        style={{ margin: '10px' }}
+      >
+        <div
+          className={`${css.gridThreeItemBox} gridThreeItemBox`}
+          style={{
+            margin: '10px', height: '100px', width: 'auto', borderRadius: '5px', backgroundColor: '#808080', textAlign: 'center', padding:'5px'
```
Comment (julianguyen) [inline client/app/stories/Styleguide.stories.jsx]: You can put the following into a shared variable remove duplicate code below:
```
{
  margin: '10px',
  height: '100px',
  width: 'auto',
  borderRadius: '5px',
  backgroundColor: '#808080',
  textAlign: 'center',
  padding:'5px', 
}
```

**Comment by KatieOfCode** [comment]
Please let me know if something looks off!

File: client/yarn.lock [hunk 12]
```diff
@@ -9,18 +9,18 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/core@^7.1.0":
```
Comment (julianguyen) [inline client/yarn.lock]: Looks like there were some merge conflicts that resulted in some packages being incorrectly downgraded. Can you delete this file locally and run `yarn install` in the client folder to generate a new file?

**Pull request merged**